dev/entrypoint.sh

```diff
@@ -5,8 +5,8 @@ set -e
 source scripts/export_datastore_variables.sh
 scripts/wait.sh $DATASTORE_WRITER_HOST $DATASTORE_WRITER_PORT
 
-printf "\nMigrations:\n"
-python openslides_backend/migrations/migrate.py finalize
-printf "\n"
+# printf "\nMigrations:\n"
```

Comment:
Why did you comment/removed this? IMO this is still needed.

Reply:
commented it out for developing purposes, added it back now